docs/tools/apiExtractorUtils.ts

```diff
@@ -0,0 +1,385 @@
+              ? [printTsDoc({ apiModel: context.apiModel, apiItem: m }, m.tsdocComment.summarySection)].flat()
+              : [];
+
+          const releaseTag = ApiReleaseTagMixin.isBaseClassOf(m) ? [strong(text(ReleaseTag[m.releaseTag]))] : [];
+
+          return tableRow([
+            tableCell(link(getFileUrl(m), '', inlineCode(displayName))),
+            tableCell([...summary, ...releaseTag]),
+          ]);
+        }),
+      ],
+    ),
+  ];
+}
+
+const getDisplayName = (
+  context: Context,
+  apiItem: ApiItem,
+  {
+    includeParams,
+    includeParamTypes,
+    includeReturnType,
+    includeReleaseTag,
+  }: { includeParams: boolean; includeParamTypes: boolean; includeReturnType: boolean; includeReleaseTag: boolean },
+) => {
+  const releaseTag =
+    includeReleaseTag && ApiReleaseTagMixin.isBaseClassOf(apiItem) ? ` (${ReleaseTag[apiItem.releaseTag]})` : '';
+
+  const functionSignature =
+    includeParams && ApiParameterListMixin.isBaseClassOf(apiItem)
+      ? getFunctionSignature(context, apiItem, { includeParamTypes, includeReturnType })
+      : '';
+
+  return apiItem.displayName + functionSignature + releaseTag;
+};
+
+const getFunctionSignature = (
+  context: Context,
+  apiItem: ApiParameterListMixin,
+  { includeParamTypes, includeReturnType }: { includeParamTypes: boolean; includeReturnType: boolean },
+) => {
+  const paramsSignature = getFunctionParamsSignature(context, apiItem, { includeParamTypes });
+  if (includeReturnType && ApiReturnTypeMixin.isBaseClassOf(apiItem)) {
+    const returnType = excerptToHtml({ apiItem, apiModel: context.apiModel }, apiItem.returnTypeExcerpt);
+    return `${paramsSignature} => ${returnType}`;
+  } else {
+    return paramsSignature;
+  }
+};
+
+const getFunctionParamsSignature = (
+  context: Context,
+  apiItem: ApiParameterListMixin,
+  { includeParamTypes }: { includeParamTypes: boolean },
+) => {
+  if (!includeParamTypes) {
+    return '(' + apiItem.parameters.map((x) => x.name).join(', ') + ')';
+  } else {
+    return (
+      '(' +
+      apiItem.parameters
+        .map((x) => `${x.name}: ${excerptToHtml({ apiItem, apiModel: context.apiModel }, x.parameterTypeExcerpt)}`)
+        .join(', ') +
+      ')'
+    );
+  }
+};
+
+function frontmatter(
+  context: Context,
+  apiItem: ApiItem,
+  fileDestination: {
+    path: string;
+    line: number;
+  } | null,
+): Literal {
+  const value = Object.entries({
+    releaseTag: ApiReleaseTagMixin.isBaseClassOf(apiItem) ? ReleaseTag[apiItem.releaseTag] : null,
+    ...(fileDestination && { fileDestination: `${fileDestination.path.replace(/^\//, '')}#L${fileDestination.line}` }),
+    title: getDisplayName(context, apiItem, {
+      includeParamTypes: false,
+      includeParams: true,
+      includeReleaseTag: true,
+      includeReturnType: false,
+    }),
+  })
+    .map(([key, val]) => `${key}: ${val}`)
+    .join('\n');
+  return { type: 'yaml', value };
+}
+
+run();
+
```

Comment:
If I would blink I might miss it, maybe it's good idea to move it to the bottom of the file :wink: